docs/content/en/docs/getting-started.md

```diff
@@ -0,0 +1,484 @@
+        ports:
+        - containerPort: 80
+---
+apiVersion: apps/v1
+kind: Service
+apiVersion: v1
+kind: Service
+metadata:
+  name: nginx
+  namespace: keptndemo
+spec:
+  selector:
+    app.kubernetes.io/name: nginx
+  ports:
+    - protocol: TCP
+      port: 8080
+      targetPort: 80
+```
+
+Now apply:
+
+```shell
+kubectl apply -f app.yaml
+```
+
+Keptn looks for these 3 labels:
+
+- `app.kubernetes.io/part-of`
+- `app.kubernetes.io/name`
+- `app.kubernetes.io/version`
+
+These are [Kubernetes recommended labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/#labels)
+but if you want to use different labels, you can swap for:
+
+- `keptn.sh/app` instead of `app.kubernetes.io/part-of`
+- `keptn.sh/workload` instead of `app.kubernetes.io/name`
+- `keptn.sh/version` instead of `app.kubernetes.io/version`
+
+## Explore Keptn
+
+Keptn is now aware of your deployments and is generating DORA statistics about them.
+
+Keptn has created a resource called a `KeptnApp` to track your application.
+The name of which is based on the `part-of` label.
+
+It may take up to 30 seconds to create the `KeptnApp` so run the following command until you see the `keptnappdemo` CRD.
+
+```shell
+kubectl -n keptndemo get keptnapp
+```
+
+Expected output:
+
+```shell
+NAME           AGE
+keptndemoapp   2s
+```
+
+Keptn also creates a new application version every time you increment the `version` label.
+
+The `PHASE` will change as the deployment progresses.
+A successful deployment is shown as `PHASE=Completed`
+
+```shell
+kubectl -n keptndemo get keptnappversion
+```
+
+Expected output:
+
+```shell
+NAME                      APPNAME        VERSION   PHASE
+keptndemoapp-0.0.1-***    keptndemoapp   0.0.1     Completed
+```
+
+Keptn can run tasks and SLO evaluations before and after deployment.
+You haven't configured this yet, but you can see the full lifecycle for a `keptnappversion` by running:
+
+```shell
+kubectl -n keptndemo get keptnappversion -o wide
+```
+
+Keptn applications are a collection of workloads.
+By default, Keptn will build `KeptnApp` CRDs based on the labels you provide.
+
+In the example above, the `KeptnApp` called `keptndemoapp` contains one workload (based on the `name` label):
+
+## View your application
+
+Port-forward to expose your app on `http://localhost:8080`:
+
+```shell
+kubectl -n keptndemo port-forward svc/nginx 8080
+```
+
+Open a browser window and go to `http://localhost:8080`
+
+You should see the "Welcome to nginx" page.
+
+![nginx demo app](../assets/nginx.png)
+
+## View DORA Metrics
+
+Keptn is generating DORA metrics and OpenTelemetry traces for your deployments.
+
+These metrics are exposed via the Keptn lifecycle operator `/metrics` endpoint on port `2222`
+
+To see these raw metrics:
+
+- Port forward to the lifecycle operator metrics service:
+
+```shell
+kubectl -n keptn-lifecycle-toolkit-system port-forward service/keptn-klt-lifecycle-operator-metrics-service 2222
```

Comment:
of course, this adds jq as a prerequisite, but maybe saves us refactoring later? 🤔

Reply:
to avoid `jq` we can use
```suggestion
SERVICE=$(kubectl get svc -l control-plane=lifecycle-operator -A -ojsonpath="{.items[0].metadata.name}")
kubectl -n keptn-lifecycle-toolkit-system port-forward svc/$SERVICE 2222
```
or even more compress
```suggestion
kubectl -n keptn-lifecycle-toolkit-system port-forward svc/$(kubectl get svc -l control-plane=lifecycle-operator -A -ojsonpath="{.items[0].metadata.name}") 2222```